versions_manager.py

```diff
@@ -13,5 +13,5 @@ def check_compatibility(self, tanner_version):
         min_version = self.version_mapper[self.version][0]
         max_version = self.version_mapper[self.version][1]
         if not (StrictVersion(min_version) <= StrictVersion(tanner_version) <= StrictVersion(max_version)):
-            print("Wrong tanner version: {}. Compatible versions are {} - {}".format(tanner_version, min_version, max_version))
+            raise RuntimeError("Wrong tanner version: {}. Compatible versions are {} - {}".format(tanner_version, min_version, max_version))
```

Comment:
if you use RuntimeError, you don't need exit. But you can also assert with SystemExit in test if you use exit()